WordPress/gutenberg
# Gallery block: Move add/edit media to block toolbar

## Description
Related to: #21247

Moves the Gallery block image add/edit from the placeholder at the bottom of the Gallery to a button in the toolbar.  

This is the first step in updating the adding of images to the Gallery. The next step will be to enable the inline block inserter, but there are currently some issues with this not working in the Gallery horizontal layout that needs to be resolved.

## How to test
Add a Gallery block and make sure that images can be added from the block placeholder, and also from the 'Add' button that appears in block toolbar once a Gallery contains images.

## Screenshots 

![gallery-toolbar](https://user-images.githubusercontent.com/3629020/149852321-34de1ea8-dcc5-4df0-8bdf-488e2fb10216.gif)
